src/ContinuumAbsorption/absorption_ff_positive_ion.jl

```diff
@@ -0,0 +1,74 @@
+   should be callables taking temperature and (photon energy / RydbergH / Zeff^2).  See 
+   [Peach1970](@ref) for details.
+
+!!! note
+    A free-free interaction is named as though the species interacting with the free electron had 
+    one more bound electron (in other words it's named as though the free-electron and ion were 
+    bound together). The `Z` argument and `ni` values should be specified for the species that 
+    actually participates in the reaction. For example:
+    - Si I ff absorption: `ni` holds the number density of Si II, and `Z=1` (net charge of Si II)
+    - Si II ff absorption: `ni` holds the number density of Si III, and `Z=2` (net charge of Si III)
+"""
+function positive_ion_ff_absorption!(α_out::AbstractVector{<:Real}, νs::AbstractVector{<:Real}, 
+                                     T::Real, number_densities::Dict, ne::Real;
+                                     departure_coefficients=Peach1970.departure_coefficients())
+    if !(_gauntff_T_bounds.lower <= T <= _gauntff_T_bounds.upper)
+        return #if T is outside the supported range return without changing α_out
+    end
+    #these are the freqs which are within the supported range
+    idx = (c_cgs/_gauntff_λ_bounds.upper) .< νs .< (c_cgs/_gauntff_λ_bounds.lower)
+
+    ndens_Z1 = 0.0
+    ndens_Z2 = 0.0
+    ndens_Z3 = 0.0
+
+    for (spec,ndens) in number_densities
+        if spec.charge <= 0
+            # skip neutral species. They don't participate in ff interations.
+            # While Korg doesn't track negatively charged ions as a separate species at the moment,
+            # skip them too, in case that changes.
+            continue 
+        elseif spec in keys(departure_coefficients)
+            D = departure_coefficients[spec]
+            # photon energy in Rydberg*Zeff^2, see equation (5) in Peach 1967 (NOT Peach 1970)
+            # https://articles.adsabs.harvard.edu/pdf/1967MmRAS..71....1P
+            σs = @. νs / spec.charge^2 * (hplanck_eV / Rydberg_eV) 
+            # add directly to α_out if there is a departure coefficient
+            @. α_out[idx] += (hydrogenic_ff_absorption(νs[idx], T, spec.charge, 
+                                                       number_densities[spec], ne)*(1+D(T, σs)))
+        else
+            #sum up contributions of hydrogenic ff coeffs, add them to α_out at the end
+            if (spec.charge == 1)     # e.g. O II
+                ndens_Z1 += ndens
+            elseif (spec.charge == 2) # e.g. O III
+                ndens_Z2 += ndens
+            else
+                error("triply+ ionized species not supported")
+            end
+        end
+    end
+
+    #add contributions from species for which we use the uncorrected hydrogenic approximation
+    @. α_out[idx] += hydrogenic_ff_absorption(νs[idx], T, 1, ndens_Z1, ne)
+    @. α_out[idx] += hydrogenic_ff_absorption(νs[idx], T, 2, ndens_Z2, ne)
+    ;
```

Comment:
It's syntactically associated with the line above, preventing it the function from returning a value.  It could be removed with no impact on the code's functioning.